src/server/routes/groups.js

```diff
@@ -110,9 +110,8 @@ router.post('/create', async (req, res) => {
 	const validGroup = {
 		type: 'object',
 		maxProperties: 4,
```

Comment:
Don't forget to change maxProperties here and on all the other validation routes you changed.